app/controllers/infrastructures_controller.rb

```
@@ -89,7 +89,8 @@ def stack_events
     events = nil
     begin
       events = stack.events
-    rescue Aws::CloudFormation::Errors::ValidationError # stack does not exist
+    rescue
+      Aws::CloudFormation::Errors::ValidationError # stack does not exist
```

Okay, I see.
But we should suppress this exception...